src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,82 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SEPARATOR = " ";
+    public static final String DELIMITER = ",";
+    public static final int NAME = 0;
+    public static final int VALUE = 1;
```

Comment:
very bad naming. has no common with task namings. dont invent your own names if they already exist
```suggestion
    public static final int OPERATION_TYPE_INDEX = 0;
    public static final int AMOUNT_INDEX = 1;
```